src/cloudai/parser.py

```diff
@@ -95,10 +95,12 @@ def parse(
         test_mapping = {t.name: t for t in tests}
         plugin_test_scenario_mapping = {}
         if PLUGIN_ROOT.exists() and list(PLUGIN_ROOT.glob("*.toml")):
+            logging.debug("PLUGIN_ROOT exists and contains .toml files. Proceeding with plugin test scenario parsing.")
```

Comment:
We also need to log case when this directory doesn't exist, but we've tried to look it up (with location).

Reply:
Please find the updated code.